app/cdap/components/ConnectionList/Components/ConnectionTabs/index.tsx

```diff
@@ -119,7 +119,7 @@ export default function ConnectionsTabs({
                     )
                   ) : (
                     <TabLabelCanBrowse
-                      label={connectorType.name}
+                      label={index === 0 ? connectorType.displayName : connectorType.name}
```

Comment:
we can change the expression to
Index === 0 || connectorType.canBrowse
<TabLabelCanBrowse />
: <TabLabelCanSample />